docs/source/guide/pec-3-options.myst

```diff
@@ -0,0 +1,414 @@
+
+The method {meth}`.OperationRepresentation.sample()` can be used for this scope:
+
+```{code-cell} ipython3
+noisy_op, sign, coeff = h_rep.sample()
+print(f"The sampled noisy operation is: {noisy_op}.")
+print(f"The associated coefficient is {coeff:g}, whose sign is {sign}.")
+```
+
+***Hint:** try re-executing the previous cell to get different samples.*
+
++++
+
+Other useful methods of {class}`.OperationRepresentation` are shown in the next cells.
+
+```{code-cell} ipython3
+# One-norm "gamma" quantifying the mitigation cost
+h_rep.norm
+```
+
+```{code-cell} ipython3
+# Quasi-probability distribution
+print(h_rep.coeffs)
+assert sum([abs(eta) for eta in h_rep.coeffs]) == h_rep.norm
+```
+
+```{code-cell} ipython3
+# Positive and normalized distribution p(alpha)=|eta_alpha|/gamma
+h_rep.distribution()
+```
+
+## Estimating expectation values with PEC
+
++++
+
+The second main step of PEC is to make use of the previously defined {class}`.OperationRepresentation`s to estimate
+expectation values with respect to a quantum state prepared by a circuit of interest.
+In the previous section we defined the representation of the Hadamard gate.
+So, for simplicity, we consider a circuit that contains only Hadamard gates.
+
++++
+
+### Defining a circuit of interest and an Executor
+
+```{code-cell} ipython3
+circuit = qiskit.QuantumCircuit(1)
+for _ in range(4):
+    circuit.h(0)
+print(circuit)
+```
+
+In this case, the list of {class}`.OperationRepresentation`s that we need for PEC is simply:
+
+```{code-cell} ipython3
+representations = [h_rep]
+```
+In general `representations` will contain as many representations as the number of ideal
+gates involved in `circuit`.
+
+**Note:** *If a gate is in `circuit` but its {class}`.OperationRepresentation` is not listed in 
+`representations`, Mitiq can still apply PEC. However, any errors associated to
+that gate will not be mitigated. In practice, all the gates without {class}`.OperationRepresentation`s 
+are treated by Mitiq as if they were noiseless.*
+
+The executor must be defined by the user since it depends on the specific frontend and backend
+(see the [Executors](guide-executors.myst) section).
+Here, for simplicity, we import the basic {func}`.execute_with_noise` function from the Qiskit utilities of Mitiq.
+
+```{code-cell} ipython3
+from mitiq import Executor
+from mitiq.interface.mitiq_qiskit import execute_with_noise, initialized_depolarizing_noise
+
+def execute(circuit):
+    """Returns Tr[ρ |0⟩⟨0|] where ρ is the state prepared by the circuit
+    executed with depolarizing noise.
+    """
+    circuit_copy = circuit.copy()    
+    noise_model = initialized_depolarizing_noise(BASE_NOISE)
+    projector_on_zero = [[1, 0], [0, 0]]
+    return execute_with_noise(circuit_copy, projector_on_zero, noise_model)
+
+# Cast the execute function into an Executor class to record the execution histroy
+executor = Executor(execute)
+```
+
+### Options for estimating expectation values
```

Comment:
I would definitely start with this section (i.e., move this section to the top), then get into the longer discussion about operation representations. That way the easiest and most relevant information is at the beginning, and you can learn more by reading more.